fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -23,8 +23,8 @@ Usage() {
   echo "             --motionmatdir=<input motion correcton matrix directory>"
   echo "             --motionmatprefix=<input motion correcton matrix filename prefix>"
   echo "             --ofmri=<input fMRI 4D image>"
-  echo "             --freesurferbrainmask=<input FreeSurfer brain mask, nifti format in T1w space>"
-  echo "             --biasfield=<input biasfield image, in T1w space>"
+  echo "             --freesurferbrainmask=<input FreeSurfer brain mask, nifti format in atlas (MNI152) space>"
```

Comment:
I don't know if this is correct.

Reply:
Here are the inputs to `OneStepResampling`, from the call in `GenericfMRIVolumeProcessingPipeline`:
```
${RUN} ${PipelineScripts}/OneStepResampling.sh \
       --workingdir=${fMRIFolder}/OneStepResampling \
       --infmri=${fMRIFolder}/${OrigTCSName}.nii.gz \
       --t1=${AtlasSpaceFolder}/${T1wAtlasName} \
       --fmriresout=${FinalfMRIResolution} \
       --fmrifolder=${fMRIFolder} \
       --fmri2structin=${T1wFolder}/xfms/${fMRI2strOutputTransform} \
       --struct2std=${AtlasSpaceFolder}/xfms/${AtlasTransform} \
       --owarp=${AtlasSpaceFolder}/xfms/${OutputfMRI2StandardTransform} \
       --oiwarp=${AtlasSpaceFolder}/xfms/${Standard2OutputfMRITransform} \
       --motionmatdir=${fMRIFolder}/${MotionMatrixFolder} \
       --motionmatprefix=${MotionMatrixPrefix} \
       --ofmri=${fMRIFolder}/${NameOffMRI}_nonlin \
       --freesurferbrainmask=${AtlasSpaceFolder}/${FreeSurferBrainMask} \
       --biasfield=${AtlasSpaceFolder}/${BiasFieldMNI} \
       --gdfield=${fMRIFolder}/${NameOffMRI}_gdc_warp \
       --scoutin=${fMRIFolder}/${OrigScoutName} \
       --scoutgdcin=${fMRIFolder}/${ScoutName}_gdc \
       --oscout=${fMRIFolder}/${NameOffMRI}_SBRef_nonlin \
       --ojacobian=${fMRIFolder}/${JacobianOut}_MNI.${FinalfMRIResolution}
```
 --freesurferbrainmask, --biasfield (and --t1) are all from `${AtlasSpaceFolder}` ("MNINonLinear").  If it is somehow incorrect to refer to those inputs as "atlas (MNI152) space" rather than "T1w space" in the usage statement, I'll change it back, but if that's that case, would like to hear from @glasserm why my change is incorrect.

Reply:
I was not claiming it was wrong, just that I didn't know, and highlighting it so someone who did know would double-check.

Reply:
Mike seems correct